controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/pricing/PricingApiHandler.java

```diff
@@ -155,12 +155,12 @@ private static SlimeJsonResponse response(PriceInformation priceInfo) {
         Cursor cursor = slime.setObject();
 
         var array = cursor.setArray("priceInfo");
-        addItem(array, "listPrice", priceInfo.listPrice());
-        addItem(array, "volumeDiscount", priceInfo.volumeDiscount());
-        addItem(array, "committedAmountDiscount", priceInfo.committedAmountDiscount());
-        addItem(array, "enclaveDiscount", priceInfo.enclaveDiscount());
+        addItem(array, "List price", priceInfo.listPrice());
+        addItem(array, "Volume discount", priceInfo.volumeDiscount());
+        addItem(array, "Committed spend discount", priceInfo.committedAmountDiscount());
```

Comment:
The item order should be the same as calculation order, so committed spend should be last? Also consider dropping "discount" here since this could be negative if the tenant under-spends.

Reply:
Yup, agreed, fixing